tests/tensor/rewriting/test_basic.py

```diff
@@ -1300,6 +1302,20 @@ def test_local_join_make_vector():
     assert check_stack_trace(f, ops_to_check="all")
 
 
+def test_local_sum_make_vector():
+    a, b, c = scalars("abc")
+    mv = MakeVector(config.floatX)
+    output = mv(a, b, c).sum()
+
+    func = function([a, b, c], output)
+
+    elemwise = func.maker.fgraph.outputs[0].owner
+    # The MakeVector op should be optimized away, so we just
+    # take the sum of the scalars.
+    assert elemwise.inputs[0].name == "a"
```

Comment:
Maybe more readable if you just check there is no MakeVector Op in fgraph.apply_nodes.